### ISSUE TITLE
SwitchCaseOnNewline triggers on enums

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#878]
I totally forgot about enums when implementing SwitchCaseOnNewlineRule.

It seems that only enums with associated values are triggering this rule:

```swift
// doesn't trigger
enum Enviromnent {
    case development
    case production
}

// triggers
enum Environment {
    case development(url: URL)
    case production(url: URL)
}
```

**Comment by jpsim** [comment]
Fixed in #877.
